src/SFA.DAS.AdminService.Web/Models/Search/LearnerDetailForStaffViewModel.cs

```diff
@@ -21,45 +15,46 @@ public class LearnerDetailForStaffViewModel
+        public string AddressedTo => GetAddressedTo(Learner);
+        public DateTime? UpdatedStatusDate => GetUpdatedStatusDate(Learner.CertificateStatus);
+        public string ReasonForChange => GetReasonForChange(Learner.ReasonForChange);
+
+        private string GetReasonForChange(string learnerReasonForChange)
+        {
+            if (string.IsNullOrWhiteSpace(learnerReasonForChange))
+                return "No reason entered";
+            return learnerReasonForChange;
+        }
+
         public string RecipientTitle => GetRecipientTitle(Learner.CertificateStatus);
-        private string GetAddressedTo(List<CertificateLogSummary> learnerCertificateLogs)
+        private string GetAddressedTo(LearnerDetailResult learnerDetailResult)
         {
-            if (learnerCertificateLogs.Any())
+            if (learnerDetailResult != null)
             {
-                var learnerLog = learnerCertificateLogs.OrderByDescending(e => e.EventTime)
-                    .FirstOrDefault(l => l.Status == Learner.CertificateStatus);
-                if (learnerLog != null)
-                {
-                    var certData = JsonConvert.DeserializeObject<CertificateData>(learnerLog.CertificateData);
-                    return certData.ContactAddLine1
-                           + Environment.NewLine
-                           + certData.ContactAddLine2
-                           + Environment.NewLine
-                           + certData.ContactAddLine3
-                           + Environment.NewLine
-                           + certData.ContactAddLine4
-                           + Environment.NewLine
-                           + certData.ContactPostCode;
-                }
+                return learnerDetailResult.ContactName
+                       + Environment.NewLine
+                       + learnerDetailResult.ContactAddLine1
+                       + Environment.NewLine
+                       + learnerDetailResult.ContactAddLine2
+                       + Environment.NewLine
+                       + learnerDetailResult.ContactAddLine3
+                       + Environment.NewLine
+                       + learnerDetailResult.ContactAddLine4
+                       + Environment.NewLine
+                       + learnerDetailResult.ContactPostCode;
             }
-
             return string.Empty;
         }
 
-        private DateTime? GetUpdatedStatusDate(List<CertificateLogSummary> learnerCertificateLogs)
+        private DateTime? GetUpdatedStatusDate(string learnerCertificateStatus)
         {
-            if (learnerCertificateLogs.Any())
-            {
-                var learnerLog = learnerCertificateLogs.OrderByDescending(e => e.EventTime)
-                    .FirstOrDefault(l => l.Status == Learner.CertificateStatus);
-                return learnerLog?.EventTime.UtcToTimeZoneTime();
-            }
-
-            return null;
+            if (learnerCertificateStatus == "SentToPrinter")
```

Comment:
This should be any of the print notification status i.e. CertificateStatus.HasPrintNotificateStatus() method